src/styles/services.module.css

```
@@ -0,0 +1,198 @@
+
+  margin-right: 0;
+  -webkit-mask-image: var(--data-uri-svg);
+  mask-image: var(--data-uri-svg);
+}
+
+/*
+ * Once a territory is picked the field turns read-only and its button resets
+ * the search, so it becomes the emphasised action of the block rather than a
+ * discreet submit.
+ */
+.heroSearchSelected :global(.fr-search-bar .fr-btn) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)